cli.js

```diff
@@ -41,9 +41,18 @@ const handleErrors = fn =>
   }
 
 const handleImportCommand = args => {
-  const { filepath, handlebarsOptionsFile, url } = args
-  let { token } = args
-  assert(fileExists(filepath), `${filepath} does not exist`)
+  const { handlebarsOptionsFile, url } = args
+  let { filepath, token } = args
+  try {
+    assert(fileExists(filepath), `${filepath} does not exist`)
+  } catch (error) {
+    const achFilepath = path.join(__dirname, './data', filepath)
```

Comment:
This sounds a bit magical. I would prefer to do it through a command line option, eg `ACH import somefile.js --use-ach-data`.
Or at the very least, document this behaviour in the import command

Reply:
script a aussi ce côté "magique" (automatique), donc j'ai fait pareil. Ca me parait bien comme comportement par défaut.

Par contre +1 pour rajouter de la doc

Reply:
Il y a pour moi une différence majeure : `ACH scripts` a un comportement unique, qui est d'aller chercher dans le répertoire ach/scripts, alors que pour `ACH import` le comportement par défaut est de prendre un fichier local dont on spécifie le path. Du coup, je ne m'attends pas à un fallback automatique sur `/data`. Et ça peut être même un peu dangereux : imagine j'ai un fichier geojson01.json en local et en faisant une typo je tape `ACH import geojson-01.json` -> ça va importer le geojson de ach/data sans que je m'en aperçoive

Reply:
`ACH scripts` n'a pas un comportement unique d'aller chercher dans le répertoire ach/scripts, on peut lui passer un script qui vient de l'app